src/query/codegen/src/writes/register.rs

```diff
@@ -736,9 +740,9 @@ pub fn codegen_register() {
 
                 format!(
                     "({arm_pat}) => {{
-                        let nullable_column = func({func_arg} ctx)?.into_column().unwrap();
+                        let nullable_column = func({func_arg} ctx).into_column().unwrap();
```

Comment:
Do we need to set `ctx.validity` here?

Reply:
if it's combine_nullable, we don't set validity to ctx. The inner function should care about the error and nulls.

Reply:
`combine_nullable` takes `Fn(I1, I2) -> NullalbeType<O>` and then returns `Fn(Nullable<I1>, Nullable<I2>) -> NullableType<O>`, therefore it does the same as `passthrough_nullable` in handling argument's nulls, and the only difference is that `combine_nullable` is able to return a NULL while all arguments are not null.